src/views/Pages/Register/Register.js

```diff
@@ -0,0 +1,245 @@
+import React, { Component } from 'react';
+import {
+  Button, Card, CardBody, Col, Container, Form, Input, InputGroup
+  , InputGroupAddon, InputGroupText, Row, Nav, NavItem, NavLink, TabContent, TabPane
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [',' should be placed last.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)